.changeset/pink-ravens-exist.md

```diff
@@ -0,0 +1,5 @@
+---
+'@builder.io/qwik-city': major
```

Comment:
this is a patch, not a major release :grimacing:

Reply:
Nice catch @wmertens 😅